src/overview/reducer.js

```diff
@@ -5,13 +5,16 @@ import * as actions from './actions'
 const defaultState = {
     searchResult: {rows: []},
     query: '',
+    isFetching: true,
 }
 
 function setQuery(state, {query}) {
+	state.isFetching = true; //will start fetching
     return {...state, query}
 }
 
 function setSearchResult(state, {searchResult}) {
+	state.isFetching = false; //will stop fetching
```

Comment:
I understand, I think the problem is around the comment. this is the state of the fetching, not something which trigger a behaviour.